docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+  reviewer-reporter:
+    build:
+      context: .
+    image: reviewer-reporter:latest
+    environment:
+      MAILER_AUTH_USER: ${MAILER_AUTH_USER}
+      MAILER_AUTH_PASS: ${MAILER_AUTH_PASS}
+      MAILER_HOST: ${MAILER_HOST}
+      MAILER_PORT: ${MAILER_PORT}
+    volumes:
+      - ./cron_tasks_folder/daily8am:/etc/periodic/daily8am/:ro
+      - ./cron_tasks_folder/daily1pm:/etc/periodic/daily1pm/:ro
+      - ./scripts/startup.sh:/usr/local/startup.sh:ro
```

Comment:
shouldn't this script inside the image? `docker-compose.yaml` is not available wherever this is deployed

Reply:
It seems to be in the image, so no need to override? Can benefit development I guess.

Reply:
Ooops - this was from when I was doing it all with docker compose - thanks!

Reply:
Removed